components/ButtonBar/src/private/MDCAppBarButtonBarBuilder.m

```diff
@@ -167,7 +178,7 @@ + (UIEdgeInsets)contentInsetsForButton:(MDCButton *)button
     return sum;
   };
 
-  const BOOL isPad = userInterfaceIdiom == UIUserInterfaceIdiomPad;
```

Comment:
Nit: I think you can leave this as const, I don't see anywhere that you change it.